src/shared/components/common/markdown-textarea.tsx

```diff
@@ -619,9 +638,16 @@ export class MarkdownTextArea extends Component<
   }
 
   getSelectedText(): string {
-    let textarea: any = document.getElementById(this.id);
-    let start: number = textarea.selectionStart;
-    let end: number = textarea.selectionEnd;
+    const { selectionStart: start, selectionEnd: end } =
+      document.getElementById(this.id) as any;
     return start !== end ? this.state.content?.substring(start, end) ?? "" : "";
   }
+
+  getIsDisabled() {
```

Comment:
Up to you, but you can add `get getIsDisable() {...` and then access it as if it were a variable: `this.getIsDisabled` 

Usually not a bad idea when there are no function params.